pkg/networks/networks.yaml

```diff
@@ -1,15 +1,19 @@
-# Paths to vde executables. Because vde_vmnet is invoked via sudo it should be
+# Paths to socket_vmnet executables. Because socket_vmnet is invoked via sudo it should be
```

Comment:
Nit: There is only a single executable:

```suggestion
# Paths to socket_vmnet executable. Because socket_vmnet is invoked via sudo it should be
```

Reply:
Fixed